cli/lib/exec/spawn.js

```diff
@@ -234,7 +235,8 @@ module.exports = {
       }
 
       return spawn(overrides)
-      .then((code) => {
+      .then((code, signal) => {
+        debug('child exited %o', { code, signal })
```

Comment:
are the code and signal always going to get values? or do we need to set defaults

Reply:
reverted this change